utils/bazel/torch-mlir-overlay/externals/llvm-external-projects/torch-mlir-dialects/test/Conversion/BUILD.bazel

```diff
@@ -0,0 +1,19 @@
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
For completeness, could you also enable the LIT tests in the `tmtensor` directory?

Reply:
Sure, done.